samples/Compute/platform-image-policy/azurepolicy.json

```diff
@@ -37,12 +50,6 @@
                                         "14.04.2-LTS"
```

Comment:
the parameters should replace these arrays.
i.e. 
`"in": ["14.04.2-LTS"] `=> `"in" : [parameters('imageSkus')]`